pkg/services/govmomi/util.go

```diff
@@ -329,7 +329,7 @@ func reconcileVSphereVMOnChannel(_ context.Context, vmCtx *capvcontext.VMContext
 					vmCtx.Logger.Error(err, "error occurred while waiting to trigger a generic event")
 				}
 				return
-			case <-vmCtx.Done():
+			case <-ctx.Done():
```

Comment:
I think we should drop this case. I don't think the idea is to cancel this go routine when the parent ctx is canceled

Reply:
@chrischdi wdyt?

Reply:
In general I think this makes sense.

I also found no special context cancelation on our side so should be good 👍

Reply:
Thanks for catching this, just created a new context here.

Reply:
Let's not create a context, remove the whole line instead.

Reply:
Fixed.